README-WAF.md

```diff
@@ -62,7 +62,18 @@ waf list
 
 ## Program groups ##
 
-The programs in ardupilot are categorized into the following groups:
+Program groups are used to represent a class of programs. They can be used to
+build all programs of a certain class without having to specify each program. A
+program can belong to more than one group.
```

Comment:
> When I read it what I though was that a program could belong to bin and tests, for example.

I think that is clarified when we read that "The main groups form a partition of all programs" below, meaning that a program don't belong to more than one **main** group.